rust/src/krb/krb5.rs

```diff
@@ -130,13 +146,26 @@ impl KRB5State {
                         self.req_id = 0;
                     },
                     12 => {
+                        let req = krb5_parser::parse_tgs_req(i);
```

Comment:
I do not know much about Kerberos.
What does this 12 mean ?
Any link to a RFC ?

Reply:
Could you create a redmine ticket to track this improvement ?

Reply:
Here it is https://redmine.openinfosecfoundation.org/issues/4950